client/client.go

```diff
@@ -194,6 +195,22 @@ func (c *client) Stop(ctx context.Context) (*rpcpb.StopResponse, error) {
 	return c.controlc.Stop(ctx, &rpcpb.StopRequest{})
 }
 
+func (c *client) AddNode(ctx context.Context, name string, execPath string, opts ...OpOption) (*rpcpb.AddNodeResponse, error) {
+	ret := &Op{}
+	ret.applyOpts(opts)
+
```

Comment:
Can we follow the pattern we have in `RestartNode` so optional fields are just set to null?

```go
	if ret.whitelistedSubnets != "" {
		req.WhitelistedSubnets = &ret.whitelistedSubnets
	}
	if ret.logLevel != "" {
		req.LogLevel = &ret.logLevel
	}
	if ret.rootDataDir != "" {
		req.RootDataDir = &ret.rootDataDir
	}
```

Reply:
done